### DISCUSSION TITLE
Wrong Github Graduation 2022 Swags 🥲

### DISCUSSION THREAD

**Discussion by sachuverma** [community/community#27430] [GitHub Education] [answered]
Today I was excited when I received letter from Github Team, which contained Github Education Graduation 2022 Swags.
But it contained wrong profile card 😕

I checked on github this person is someone else, right card has correct username, but this left card is somebody else’s.

![D1456B0B-8E8C-4D1C-B00E-C9354F1EE52E](https://user-images.githubusercontent.com/43083013/181448709-53e3e6b5-86f6-479f-80d6-ec379f14510e.jpeg)

I know those handling this are also humans and mistakes can happen with anyone, but still why was it me. 😅

**Comment by lasr21** [comment, marked as answer]
There's been a mistake with trading cards shipped within the United States. If you are in the US, you likely received the wrong card.

Our team is reprinting cards and getting them in the mail as soon as possible. You do not need to send the card you received anywhere.

Thanks for your patience and understanding 💖

**Comment by JasmineKc** [comment]
Thanks For Support
